addon/-private/extensions/with-validation.js

```diff
@@ -21,10 +24,17 @@ export function withExtension(klass) {
     init(...args) {
       super.init(...args);
 
-      const validations = getValidationsFor(this.constructor);
+      const { constructor } = this;
+
+      const validations = getValidationsFor(constructor);
+      const meta = Ember.meta(this);
 
       for (let key in validations) {
-        wrapField(this.constructor, this, validations, key);
+        const validation = validations[key];
+
+        wrapComputedProperty(constructor, this, meta, validation, key);
```

Comment:
This no-ops if it's not a computed property; the only case where anything is really done is if a subclass is overriding a property defined with `@argument` with a computed property.

Reply:
Closing, as this is probably just noise to anyone reviewing.